### PR TITLE
Kill a usage of cpu_zero (baddbmm).

### PR DESCRIPTION
**Pull request by gchanan**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#27981 Kill a usage of cpu_zero (baddbmm).**

Similar to https://github.com/pytorch/pytorch/pull/26348/, but only a single change because I don't know why that PR is failing.

Differential Revision: [D17931058](https://our.internmc.facebook.com/intern/diff/D17931058)

### CODE REVIEW

**Comment by gchanan** [comment]
See #26348 for an explanation of the situation.

File: aten/src/ATen/Declarations.cwrap [hunk 1]
```diff
@@ -1515,7 +1515,6 @@
     - arg: THTensor* result
       output: True
       resize: [ [self,0], [self,1], [mat2,2] ]
-      cpu_zero: True
```
Comment (zou3519) [inline aten/src/ATen/Declarations.cwrap]: What is `cpu_zero` supposed to do?

Comment (gchanan) [reply aten/src/ATen/Declarations.cwrap]: For the out-of-place variant, it zeros the output that is passed in to the TH function if on CPU.

(Yes, we have a flag for that).

**Review by zou3519 — APPROVED** [review]

**Comment by gchanan** [comment]
Superseded by https://github.com/pytorch/pytorch/pull/27980.

**Pull request closed without merging**